### ISSUE TITLE
runtimeConfig changes the last 4 values ​​of an 18-number ID

### ISSUE THREAD

**Issue by Danny2105** [nuxt/nuxt.js#8317]
### Versions

- nuxt: v2.14.7
- node: v13.14.0

### Reproduction
I am using [runtime config](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-runtime-config) to share certain values ​​between client and server when using this value (the ID) in a plugin (`~/plugins /mi-plugin.js`) the value of the ID changes but only the last 4 numbers.

`nuxt.config.js`
```js
export default {
 modules: ['@nuxt/http'],
 publicRuntimeConfig: {
  clientID: process.env.CLIENT_ID
 },
 plugins: ['~/plugins/api']
}
```

`~/plugins/api.js`
```js
import { DashboardApi } from '~/lib/api';

export default function({ $config }, inject){
	console.log('$config:', $config.clientID);
	console.log('process.env:', process.env.CLIENT_ID);

	inject('api', new DashboardApi({
		clientID: $config.clientID
	}));
}
```

`package.json`
```json
{
    "name": "dashboard",
    "version": "0.1.0",
    "description": "Dashboard",
    "author": "Danny",
    "license": "MIT",
    "private": true,
    "scripts": {
        "dev": "nuxt",
        "build": "nuxt build",
        "generate": "nuxt generate",
        "start": "nuxt start"
    },
    "dependencies": {
        "@nuxt/http": "^0.6.0",
        "nuxt": "^2.14.7"
    }
}
```
### Steps to reproduce
- Open the terminal in the work area
- run the command: `yarn dev`

### What is Expected?
The correct functioning of the API plugin

### What is actually happening?
The API plugin gives me an unknown application error since in `$config.clientID` it is not the ID

Image of the two `console.log()`:
![Terminal](https://i.ibb.co/rb0p3yN/Screenshot-2.png)

**Comment by danielroe** [comment]
It looks like a rounding error, possibly because the runtime config is treating the id as a number (via destr), not a string.

**Comment by dtmzr** [comment]
Can confirm this behavior/bug.

@danielroe are you having a look into it?
Else I think this could be something I might be able to figure out and try to fix.

**Comment by Danny2105** [comment]
> It looks like a rounding error, possibly because the runtime config is treating the id as a number (via `destr`), not a string.
> 
> For now, a workaround should be to add a letter in there and you should stop seeing this behaviour.

Thanks for the reply. If when adding a letter you obviously treat it as a string and this behavior no longer occurs.

The only thing I don't see practical is to remove the letter in all the places where I need the identification, but for a quick fix that's fine.

**Comment by danielroe** [comment]
@dtmzr Go for it 👍

**Comment by pi0** [comment]
Thanks @Danny2105 for reporting and @danielroe for hinting about issue. Should be fixed with [destr@1.0.1](https://github.com/nuxt-contrib/destr/releases/tag/v1.0.1). Using `yarn upgrade destr` (or recreate lock file) should fix the problem.
